packages/workers/communities/src/index.ts

```diff
@@ -53,6 +54,9 @@ router.get('/getMembers/:communityId/:offset', ({ params }, env) =>
 router.get('/getIsMember/:communityId/:profileId', ({ params }, env) =>
   getIsMember(params.communityId, params.profileId, env)
 );
+router.get('/getCommunitySlugs/:slug', ({ params }, env) =>
```

Comment:
Rename to /search

Also `:slug` should be a query and not path just for standards, replacing URL is costly than updating the query

Eg: /search?query=lenster